### PR TITLE
ZEB-781/ZEB-848 fix browser/platform version display

### PR DESCRIPTION
**Pull request by MitskevichVS**

### CODE REVIEW

File: client/app/_services/tests-sessions/tests-sessions.service.js [hunk 1]
```diff
@@ -22,11 +22,16 @@ const testsSessionsService = function testsSessionsService(
...
         fetchAdditionalSearchParams,
         getSessionById,
+        refactorPlatformData,
 
         get activeParams() { return lastParams; },
         set activeParams(newSC) { lastParams = newSC; return true; },
     };
 
+    function refactorPlatformData(data) {
+        return [data.browserName.toLowerCase(), data.version, data.osName.toLowerCase()];
```
Comment (danymill) [inline client/app/_services/tests-sessions/tests-sessions.service.js]: Please, do not convert to lower case `osName`

File: client/app/containers/test-run-info/test-run-info.controller.js [hunk 2]
```diff
@@ -4,10 +4,10 @@ import ImagesViewerController from '../../components/modals/images-viewer/images
...
 const testRunInfoController = function testRunInfoController($scope, $rootScope, $mdDialog, $interval,
     $filter, $anchorScroll, $location, $timeout, $q,
-    elasticsearchService, TestRunService, UtilService,
-    ArtifactService, $stateParams, OFFSET,
-    API_URL, $state, TestRunsStorage,
-    TestService, $transitions, pageTitleService) {
+    elasticsearchService, testsRunsService, UtilService,
+    ArtifactService, $stateParams, OFFSET, API_URL,
+    $state, TestRunsStorage, TestService, $transitions,
+    pageTitleService) {
```
Comment (danymill) [inline client/app/containers/test-run-info/test-run-info.controller.js]: Please split to the one column and add the trailing comma;
And please double-check dependencies: if nothing is missed and there are no unused ones.

File: client/app/modules/test-session-logs/test-session-logs.controller.js [hunk 3]
```diff
@@ -39,6 +39,7 @@ const testSessionLogsController = function testsSessionsController(
     return vm;
 
     function init() {
+        [vm.testSession.platformIcon, vm.testSession.platformVersion, vm.testSession.osName] = testsSessionsService.refactorPlatformData(vm.testSession);
```
Comment (danymill) [inline client/app/modules/test-session-logs/test-session-logs.controller.js]: We don't need to convert to lower case `osName`, so remove it from this refactoring action

File: client/app/modules/tests-sessions/test-session-card/test-session-card.controller.js [hunk 4]
```diff
@@ -45,6 +46,7 @@ const testsSessionCardController = function testsSessionCardController(
     }
 
     function onInit() {
+        [vm.testSession.platformIcon, vm.testSession.platformVersion, vm.testSession.osName] = testsSessionsService.refactorPlatformData(vm.testSession);
```
Comment (danymill) [inline client/app/modules/tests-sessions/test-session-card/test-session-card.controller.js]: See comment above

**Review by danymill — CHANGES_REQUESTED** [review]

File: client/app/_services/tests-sessions/tests-sessions.service.js (hunk 1 above)
Comment (MitskevichVS) [reply client/app/_services/tests-sessions/tests-sessions.service.js]: fixed

File: client/app/modules/test-session-logs/test-session-logs.controller.js (hunk 3 above)
Comment (MitskevichVS) [reply client/app/modules/test-session-logs/test-session-logs.controller.js]: fixed

File: client/app/modules/tests-sessions/test-session-card/test-session-card.controller.js (hunk 4 above)
Comment (MitskevichVS) [reply client/app/modules/tests-sessions/test-session-card/test-session-card.controller.js]: fixed

File: client/app/containers/test-run-info/test-run-info.controller.js (hunk 2 above)
Comment (MitskevichVS) [reply client/app/containers/test-run-info/test-run-info.controller.js]: fixed

**Review by danymill — APPROVED** [review]

**Pull request merged**